uber/templates/base.html

```diff
@@ -125,6 +125,38 @@
           }
         }
     </style>
+
+    {% block feedback_javascript %}
```

Comment:
... so uh, @RobRuana, is feeding any event's instance of RAMS into MAGFest's JIRA intended behavior?

Reply:
I feel like we want an answer to this before we pull this in.